File: C/Desenvolvimento_Cc/fatorial.c
#include<stdio.h>

int main(){
    int i;
    double f;

    for(i = 1, f = 1; i<=30; i++){
        printf("fatorial de %2d = %0.0f\n", i, f);
        f += f*i;
    }
    return 0;
}

File: Python_DSA/Introdução/Cap02/fatorial.py
#fatorial de um número digitado pelo usuário
n = int(input("digite um número: ")) #lendo o número
f = 1 #fatorial inicial
for c in range(n, 0, -1): #iniciando o loop
    f *= c #fatorial 
print(f) #imprimindo o fatorial